app/models/answer.rb

```diff
@@ -9,6 +9,12 @@ class Answer < ActiveRecord::Base
 
   validates :body, :question_id, :author_id, presence: true
   validates :body, length: { minimum: 2 }
+  validate  :question_not_closed
 
   scope :recent, -> { order('created_at DESC') }
+
+  def question_not_closed
+    errors.add(:question_closed, "The question you were trying to answer is older than 30 days") if
+      !question_id.blank? and question.closed
```

Comment:
Well, I suppose the only way it wouldn't be there is if someone is doing something they shouldn't. I can remove it if you want but I was thinking better safe than sorry.